Validation/HGCalValidation/interface/validHit.h

```diff
@@ -0,0 +1,43 @@
+#ifndef _validHit_h_
```

Comment:
Include guard should contain the full path of the file, e.g.
```suggestion
#ifndef Validation_HGCalBValidation_validHit_h
```